src/classes/CW_AudienceGroupCompControlerTest.cls

```diff
@@ -0,0 +1,270 @@
+/**
+* File Name   :    CW_AudienceGroupCompControlerTest.cls
+* Description :   
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 03/04/2014 Cristina P		Initial
+*/
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.